commands/game.py

```diff
@@ -0,0 +1,935 @@
+                85665230, 86594160, 87528210, 88467380, 89411670, 90361080, 91315610, 92275260, 93240030, 94209920,
+                95184930, 96165060, 97150310, 98140680, 99136170, 100136780, 101142510, 102153360, 103169330, 104190420,
+                105216630, 106247960, 107284410, 108325980, 109372670, 110424480, 111481410, 112543460, 113610630,
+                114682920, 115760330, 116842860, 117930510, 119023280, 120121170, 121224180, 122332310, 123445560,
+                124563930, 125687420, 126816030, 127949760, 129088610, 130232580, 131381670, 132535880, 133695210,
+                134859660, 136029230, 137203920, 138383730, 139568660, 140758710, 141953880, 143154170, 144359580,
+                145570110, 146785760, 148006530, 149232420, 150463430, 151699560, 152940810, 154187180, 155438670,
+                156695280, 157957010, 159223860, 160495830, 161772920, 163055130, 164342460, 165634910, 166932480,
+                168235170, 169542980, 170855910, 172173960, 173497130, 174825420, 176158830, 177497360, 178841010,
+                180189780, 181543670, 182902680, 184266810, 185636060, 187010430, 188389920, 189774530, 191164260,
+                192559110, 193959080, 195364170, 196774380, 198189710, 199610160, 201035730, 202466420, 203902230,
+                205343160, 206789210, 208240380, 209696670, 211158080, 212624610, 214096260, 215573030, 217054920,
+                218541930, 220034060, 221531310, 223033680, 224541170, 226053780, 227571510, 229094360, 230622330,
+                232155420, 233693630, 235236960, 236785410, 238338980, 239897670, 241461480, 243030410, 244604460,
+                246183630, 247767920, 249357330, 250951860, 252551510, 254156280, 255766170, 257381180, 259001310,
+                260626560, 262256930, 263892420, 265533030, 267178760, 268829610, 270485580, 272146670, 273812880,
+                275484210, 277160660, 278842230, 280528920, 282220730, 283917660, 285619710, 287326880, 289039170,
+                290756580, 292479110, 294206760, 295939530, 297677420, 299420430, 301168560, 302921810, 304680180,
+                306443670, 308212280, 309986010, 311764860, 313548830, 315337920, 317132130, 318931460, 320735910,
+                322545480, 324360170, 326179980, 328004910, 329834960, 331670130, 333510420, 335355830, 337206360,
+                339062010, 340922780, 342788670, 344659680, 346535810, 348417060, 350303430, 352194920, 354091530,
+                355993260, 357900110, 359812080, 361729170, 363651380, 365578710, 367511160, 369448730, 371391420,
+                373339230, 375292160, 377250210, 379213380, 381181670, 383155080, 385133610, 387117260, 389106030,
+                391099920, 393098930, 395103060, 397112310, 399126680, 401146170, 403170780, 405200510, 407235360,
+                409275330, 411320420, 413370630, 415425960]
+
+    return xp_table
+
+
+def get_xp_per_flame(flames_used: int):
```

Comment:
hould just use a match case for this i.e.

```suggestion
def get_xp_per_flame(flames_used: int):
    match flames_used:
        case 1:
            return 2500
        case 2:
            return 3000
        case 3:
            return 3500
```